fwoa-tools/src/migrationImport.ts

```diff
@@ -73,9 +89,61 @@ async function startImport(folderNames: string[]): Promise<void> {
           .promise();
         if (jobStatus.ImportJobProperties.JobStatus === 'COMPLETED') {
           console.log(`successfully imported folder ${folderName}`);
-          // check each file
+          // use outputFile to check
+          const path = jobStatus.ImportJobProperties.JobOutputDataConfig?.S3Configuration?.S3Uri.replace(
+            's3://',
+            ''
+          ).replace(`${IMPORT_OUTPUT_S3_BUCKET_NAME!}/`, '');
+          const s3Client = new S3({
+            region: API_AWS_REGION!
+          });
+          const fhirClient = await (smartClient ? getFhirClientSMART() : getFhirClient());
+          const healthLakeClient = axios.create();
+          const interceptor = aws4Interceptor({
+            region: API_AWS_REGION!,
+            service: 'healthlake'
+          });
+          healthLakeClient.interceptors.request.use(interceptor);
+          // eslint-disable-next-line security/detect-object-injection
+          for (let j = 0; j < outputFile.itemNames[folderName].length; j += 1) {
+            // eslint-disable-next-line security/detect-object-injection
+            const resourcePath = outputFile.itemNames[folderName][j].replace(
+              outputFile.jobId,
+              `${path}SUCCESS`
+            );
+            const resourceFile = await s3Client
+              .getObject({
+                Bucket: IMPORT_OUTPUT_S3_BUCKET_NAME!,
+                Key: resourcePath
+              })
+              .promise();
+            if (resourceFile.$response.error) {
+              throw new Error(`Failed to read file ${resourceFile.$response.error}`);
+            }
+            const allResourceVersions: string[] = resourceFile.Body!.toString().split('\n');
+            let resource = JSON.parse(allResourceVersions[allResourceVersions.length - 1]); // latest version
+            console.log(resource);
+            // eslint-disable-next-line security/detect-object-injection
+            const responseKey = Object.keys(resource).find((x) => resource[x].jsonBlob !== undefined);
+            resource = resource[responseKey!].jsonBlob;
+            if (resource.meta.tag.length >= 2) {
+              // DELETE the resource from HealthLake
```

Comment:
where is tag length set? is there a more clear flag for which resources are marked for deletion?